src/main/java/com/delphix/masking/initializer/maskingApi/BackupDriver.java

```diff
@@ -239,6 +247,32 @@ private void finishBackup() {
         }
     }
 
+    private boolean isBackupRequired(int inputMajorVersion, int inputMinorVersion, int inputMicroVersion) {
```

Comment:
I'm not sure about the name of this function. What we are really doing is checking if the version of the engine is greater than or equal to that of the inputted version, it doesn't actually have anything to do with backing up, we are just using the result of this check to make that decision.

I suggest changing this to `isEngineVersionAtLeast` or `isEngineVersionGreaterThanOrEqualTo`. This will allow the use of this function to make sense in other contexts as well.

Reply:
Done.